server/odc-service/src/main/java/com/oceanbase/odc/service/queryprofile/display/SqlPlanGraphData.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.queryprofile.display;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.Data;
+
+/**
+ * @author liuyizhuo.lyz
+ * @date 2024/4/11
+ */
+@Data
+public class SqlPlanGraphData {
+    private List<GraphEdgeData> edgeList;
```

Comment:
field naming: edges/vertexes

topNodes may use List, order by cost

Reply:
The reason `topNodes` uses a map is that there is more than one sorting dimension, which will be sorted based on CPU usage, memory usage, and time usage.
And the key is the dimension.

Reply:
renamed.